### DISCUSSION TITLE
A GitHub Tutorial for Everyone! - Saturday, 19/10/2024 12Pm-2Pm UTC!

### DISCUSSION THREAD

**Discussion by Rayccheal** [clentafrica/hr-people-operations#14] [Announcements]
Hello Team members, @clentafrica/wg-strategy @clentafrica/wg-sustainability-esg @clentafrica/wg-design @clentafrica/wg-disability-gender-equality @clentafrica/wg-education @clentafrica/wg-energy @clentafrica/wg-hr @clentafrica/wg-research-and-data @clentafrica/wg-programs @clentafrica/wg-web-dev @clentafrica/wg-fundraising @clentafrica/wg-community-managers @clentafrica/wg-partnerships @Aarady-sharma @EmmanetD @Onodjayeke @yigakpoa 

As an offshoot of our previous team leads meeting and recognizing that many of us may not be familiar with navigating GitHub—an important component of our operational framework at CLENT Africa—the Human Resource team @clentafrica/wg-hr in collaboration with the Web Development team @clentafrica/wg-web-dev - @Aarady-Sharma @EmmanetD, are excited to announce that there would be a live GitHub tutorial this Saturday, 19/10/2024, from 12 PM to 2 PM UTC!

**Key topics to be covered are:**

- Setting up repositories
- Collaborating with branches
- Pull requests, code reviews, and much more!

Whether you are new to GitHub or aiming to enhance your skills, this tutorial is perfect for you!

**Meeting Details:**

**Date**: 19/10/2024
**Time**: 12 PM - 2 PM UTC (Check with your local time zone for clarifications [here](https://www.worldtimebuddy.com/))
**Venue**: Zoom
**Zoom Link**: [Join the meeting](https://us05web.zoom.us/j/84603494354?pwd=kL3iVW3BoyYpbtzedaK6yDDiar00Ue.1) 

P.S.: All Team Heads should kindly engage team members to ensure a maximum turnout, especially for those with little or no knowledge of GitHub.

Thank you, and I look forward to learning together with all of you soon!

Regards,
Racheal
For HR Team Coordinators

**Comment by Rayccheal** [comment]
Hello everyone!

I hope you’re all doing well!

Just a quick reminder that our live GitHub tutorial is happening tomorrow from 12-2 PM UTC. This session is a fantastic opportunity for both newcomers and experienced members to get familiar with GitHub. Your participation is important, especially for those who are new, and are having difficulty finding their way around our community.

@Aaradhy-Sharma, could you please confirm your availability for the tutorial?

We’re excited and looking forward to having you with us!

Best regards,
Racheal

**Reply by chibuikeemmanuelh** [reply]
I will be available!

**Reply by Rayccheal** [reply]
Hello @emmanuelhenryc , can you kindly confirm if you can take up the GitHub tutorial?

@Aaradhy-Sharma is yet to confirm attendance, and we'd like to know if someone else is available to take up the tutorial in case he would not be available. 

Thank you as I anticipate a response soonest. 

Kind Regards,
Racheal

**Reply by Aaradhy-Sharma** [reply]
Greetings, I would be available for the same, however our University internet connection has been unreliable for the past couple of days.
Regards

**Reply by Rayccheal** [reply]
Hi @Aaradhy-Sharma!!

I'm so sorry about your internet connection, and all the inconveniences it may have caused you.

I think what we can do is to connect, and hope that the internet is stable today? While we hope that someone may be available to take up in case it falters. Or, what do you think?

**Reply by Aaradhy-Sharma** [reply]
Understood, that would be great!
Meanwhile, I will try to connect to a more stable ethernet.

Regards

**Reply by chibuikeemmanuelh** [reply]
Hi @Rayccheal, I would be attending to learn, I wish I was well grounded in navigating and using Github so as to take the tutorial.

**Comment by MadhalasaSJ** [comment]
I will be available

**Reply by Rayccheal** [reply]
Look forward to seeing you @MadhalasaSJ !🚀🚀🚀

**Comment by Rayccheal** [comment]
Hello @everone!!

Please join the tutorial here: https://us05web.zoom.us/j/84603494354?pwd=kL3iVW3BoyYpbtzedaK6yDDiar00Ue.1

**Comment by Rayccheal** [comment]
Hello @everyone!

Thank you so much for your time - for joining the tutorial, and for kindly taking the attendance. We sincerely apologize for the turn out of events. 

We are looking at better options to bringing you a tutorial soon, that we hope will not be as inconveniencing, but for the meantime, thank you for your patience, and understanding. 

Best Regards,
Racheal 
For HR Team Coordinators
